packages/react-composites/tests/browser/common/testAppUtils.tsx

```diff
@@ -38,7 +39,8 @@ export function initializeIconsForUITests(): void {
       Link: <Link16Regular />,
       People: <People16Regular />,
       PeopleAdd: <PeopleAdd16Regular />,
-      Warning: <Warning16Regular />
+      Warning: <Warning16Regular />,
+      ProtectedDocument: <DocumentBriefcase20Regular />
```

Comment:
Perhaps [document error](https://iconcloud.design/search/filter/Fluent%20System%20Library/Fluent%20Regular/document/6e1d50a66-3821b375b) or [document lock](https://iconcloud.design/search/filter/Fluent%20System%20Library/Fluent%20Regular/document/4d930d5c2-3821b375b) might be more applicable than document briefcase here?

Reply:
Done